config/01-ce-auth.conf

```diff
@@ -7,22 +7,6 @@
 #
 ###############################################################################
 
-# Uncomment the following lines if your SciTokens or SSL clients use
-# grid certificates or your HTCondor-CE's host certificate is located
-# in the standard grid location
-#
-# https://htcondor-ce.readthedocs.io/en/latest/installation/htcondor-ce#configuring-certificates
-
-# AUTH_SSL_SERVER_CERTFILE = /etc/grid-security/hostcert.pem
-# AUTH_SSL_SERVER_KEYFILE = /etc/grid-security/hostkey.pem
-# AUTH_SSL_SERVER_CADIR = /etc/grid-security/certificates
-# AUTH_SSL_SERVER_CAFILE =
```

Comment:
I would actually prefer keeping this. We still want to use the OS CAs by default when we only have the option of picking one or the other instead of ranking them

Reply:
With this configuration, the code should consider both the OS CAs and the grid CAs. You can provide both a CA file and a CA directory to OpenSSL, and it should look in both for any request.

Reply:
Oh, that's good! What wins if there's a collision, i.e. Let's Encrypt in both the CA file + CA dir?

Reply:
The question was more of a curiosity -- we don't actually know what happens -- but I don't think we should hold up this change for my question!